test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/TypeEnumeratorTests.cs

```diff
@@ -630,5 +628,18 @@ public new void DerivedTestMethod()
         }
     }
 
+    internal abstract class DisplayNameTestClass
+    {
+        [TestMethodV2]
+        public void TestMethodWithoutDisplayName()
+        {
+        }
+
+        [TestMethodV2("Test method display name.")]
```

Comment:
What about adding more special characters in the test? `"Test method display name.  Second sentence - 'single quote' `` - 😀😍 ; ( ) & ^ # $ / \ . ,"`

Reply:
Yeah, I'll try to do that, let's see how TestExplorer likes emojis 😁 Because what would be the ability to provide custom test names without being able to use emojis!

Reply:
[We need emoji support in VS 😉](https://developercommunity.visualstudio.com/idea/734269/visual-studio-2019-add-emojis-support.html)